modules/administration-guide/partials/proc_adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance.adoc

```diff
@@ -17,12 +17,12 @@ pass:[<!-- vale RedHat.TermsErrors = YES -->]
 
 include::example$snip_{project-context}-steps-to-add-or-remove-extensions-in-openvsx-sync-json.adoc[]
 
-** If the extension is published on link:https://open-vsx.org[open-vsx.org], you can add its extension `id` in the format __<published_by>__.__<unique_identifier>__. You can find the `id` details on the extension's listing page on link:https://open-vsx.org[open-vsx.org].
+** If the extension is published on link:https://open-vsx.org[open-vsx.org], you can copy and use the *Unique identifier* from the extension's listing page on link:https://open-vsx.org[open-vsx.org]:
```

Comment:
Consider skipping the second mention of the vsx web address since it's mentioned earlier in the same sentence already.

Reply:
Thank you. That's great. I'm incorporating this in my update.